### PR TITLE
Test files for GNSSROS process in GEOS.

### PR DESCRIPTION
**Pull request by gmao-jjin3**
Testing files for GNSSRO processing in latest GEOS experiment (internally named x0048).

data needed for[ UFO PR2872](https://github.com/JCSDA-internal/ufo/pull/2872)

### CODE REVIEW

File: testinput_tier_1/gnssro_bend_geoval_2022061500_m.nc4
Comment (PatNichols) [inline testinput_tier_1/gnssro_bend_geoval_2022061500_m.nc4:1]: This file looks a little large (1.97 MB). Is it possible to shrink this somewhat? Maybe >= 250 kb?

**Comment by gmao-jjin3** [comment]
@gmao-yzhu  Can you approve this PR for uploading GNSSRO data files? Thanks.

File: testinput_tier_1/gnssro_bend_geoval_2022061500_m.nc4
Comment (PatNichols) [reply testinput_tier_1/gnssro_bend_geoval_2022061500_m.nc4:1]: @gmao-jjin3 thanks!

**Review by PatNichols — APPROVED** [review]
Thanks for shrinking the file size.

**Comment by gmao-yzhu** [comment]
I approve this PR. Sorry for being late, I have been on travel.

**Review by gmao-yzhu — APPROVED** [review]

**Comment by gmao-jjin3** [comment]
No problem at all. Thank you @gmao-yzhu .

**Comment by gmao-jjin3** [comment]
@BenjaminRuston  @PatNichols  It would be great that we have these data files merged into develop branch.

**Comment by BenjaminRuston** [comment]
@gmao-jjin3 had not seen this is there a ctest using these files or this is a precursory step?

if so not a bad idea to have these in place first

**Comment by gmao-jjin3** [comment]
@BenjaminRuston  This is a ctest using these files. Please see
https://github.com/JCSDA-internal/ufo/pull/2872/files#diff-41f683c86f883157a46055bf2ef5889a80ee101b5d1a3334a6a21f2f3bce50b4R100

**Comment by huishao-r** [comment]
> @BenjaminRuston @PatNichols It would be great that we have these data files merged into develop branch.

This one has to be merged with UFO PR together. So we need to have UFO PR ready first.

**Comment by BenjaminRuston** [comment]
@gmao-jjin3 yes know about [UFO PR2872](https://github.com/JCSDA-internal/ufo/pull/2872), and that PR should be using this ufo-data branch of the same name.  is this not working, usually merge them at the same time once the UFO PR is ready

**Review by fcvdb — APPROVED** [review]
Thanks Jianjun!

**Pull request merged**